app/assets/javascripts/notes_feed/NotesFeedPage.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+const styles = {
+  button: {
+    marginTop: '10px',
+    display: 'inline'
+  },
+  feed: {
+    margin: 'auto 10%'
+  },
+  footer: {
+    margin: '5% 41%'
+  },
+  header: {
+    margin: '2.5% 5%'
+  },
+  subTitle: {
+    fontSize: '18px',
+    marginTop: '5px'
+  },
+  title: {
+    fontSize: '28px',
+    fontWeight: 'bold',
+    color: '#155094'
+  },
+};
+
+class NotesFeedPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const NotesList = window.shared.NotesList;
+    const feed = {
+      event_notes: this.props.eventNotes,
+      deprecated: {
+        interventions: []
+      },
+      services: {
+        active: [],
+        discontinued: []
+      }
+    };
+
+    return (
+      <div className="wrapper">
+        <div className="header" style={styles.header}>
+          <div className="title" style={styles.title}>
+            My Notes
+          </div>
+          <p style={styles.subTitle}> Notes you created in the past 30 days. </p>
+        </div>
+        <div className="feed" style={styles.feed}>
+          <div className="notes-list">
+            <NotesList
+              educatorsIndex={this.props.educatorsIndex}
+              eventNoteTypesIndex={this.props.eventNoteTypesIndex}
+              feed={feed} />
+          </div>
+        </div>
+        <div className="footer" style={styles.footer}>
+          <button
```

Comment:
Talking this out with @gabivoicu — maybe we only want this to load if educator has notes / has notes in past 30 days.